client/my-sites/sidebar/index.jsx

```diff
@@ -1149,12 +1152,8 @@ function mapStateToProps( state ) {
 		isAtomicSite: !! isSiteAutomatedTransfer( state, selectedSiteId ),
 		isMigrationInProgress,
 		isVip: isVipSite( state, selectedSiteId ),
-		showCustomizerLink:
-			! (
-				isSiteUsingFullSiteEditing( state, selectedSiteId ) ||
-				isSiteUsingCoreSiteEditor( state, selectedSiteId )
-			) && siteId,
-		showSiteEditor: isSiteUsingCoreSiteEditor( state, selectedSiteId ),
+		showCustomizerLink: ! isSiteUsingFullSiteEditing( state, selectedSiteId ) && siteId,
+		showSiteEditor: isFSEActive,
```

Comment:
😆  at myself for the immediate confusion when looking at `isFSEActive` vs. `isSiteUsingFullSiteEditing` and forgetting the former is legacy-FSE.  I wonder if we should just change the name of this selector completely to something like `isSiteUsingLegacyFSE`.

Reply:
All the more reason to migrate legacy FSE sites sooner rather than later--even so, if you changed the selector names now I will approve it!

Reply:
Ive made a note, and will do that at some point after this is merged.